From: Jan Engelhardt <jengelh@inai.de>
Date: 2012-11-16 01:01:33.597645291 +0100

Tweak the build script so that we can actually _pass_ some flags in.

---
 build |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: ohcount/build
===================================================================
--- ohcount.orig/build
+++ ohcount/build
@@ -17,7 +17,7 @@ then
   fi 
 
   # You shouldn't have to change the following.
-  CFLAGS=-O3
+  CFLAGS="-O3 $CFLAGS"
   WARN="-Wall -Wno-pointer-to-int-cast -Wno-parentheses"
   SHARED=-shared
   SHARED_NAME=libohcount.so
@@ -28,7 +28,7 @@ else
   INC_DIR=/opt/local/include
   LIB_DIR=/opt/local/lib
   # You shouldn't have to change the following.
-  CFLAGS="-fno-common -g"
+  CFLAGS="-fno-common -g $CFLAGS"
   WARN="-Wall -Wno-parentheses"
   SHARED="-dynamiclib -L$LIB_DIR -lpcre"
   SHARED_NAME=libohcount.dylib
